operations/index.md

```diff
@@ -131,6 +131,8 @@ Use the Boot Orchestration Service \(BOS\) to boot, configure, and shut down col
 Procedures required for a full power off of an HPE Cray EX system.
 
   * [System Power Off Procedures](power_management/System_Power_Off_Procedures.md)
+
+Additional links to *power off sub procedures* provided for reference:
```

Comment:
Instead of adding this line, we can just indent the bullets under this line. This is how the other sections of the index have shown subtopics.

Reply:
Fixed and updated files.